src/Error/Renderer/WebExceptionRenderer.php

```diff
@@ -104,28 +99,6 @@ class WebExceptionRenderer implements ExceptionRendererInterface
      */
     protected ?ServerRequest $request;
 
-    /**
-     * Map of exceptions to http status codes.
-     *
-     * This can be customized for users that don't want specific exceptions to throw 404 errors
-     * or want their application exceptions to be automatically converted.
-     *
-     * @var array<string, int>
-     * @psalm-var array<class-string<\Throwable>, int>
-     */
-    protected array $exceptionHttpCodes = [
```

Comment:
We could end up breaking userland exception rendering with this change. If we wanted to avoid the breakage, we could retain this as an empty list and also retain line 408 as a fallback for exceptions that don't implement `HttpErrorCodeInterface`

Reply:
Okay, I'll add it back for BC.